blockchain/pool.go

```diff
@@ -29,10 +29,10 @@ eg, L = latency = 0.1s
 */
 
 const (
-	requestIntervalMS         = 100
-	maxTotalRequesters        = 1000
+	requestIntervalMS         = 2
```

Comment:
This will effectively mean no sleep in requestRoutine.

Reply:
That's OK, it just needs to prevent a hot loop.